lib/payrix/resource/base.rb

```diff
@@ -115,9 +115,14 @@ def create(params = {})
         
         headers = build_headers
         headers['Content-Type'] = "application/json"
+        query_params = []
+        if request_options
+          query_params << request_options.expand if request_options.expand
+        end
+
         method = 'post'
         url = Payrix.configuration.url
-        endpoint = "#{@resource_name}"
+        endpoint = "#{@resource_name}?#{query_params.join('&')}"
```

Comment:
This is the same approach copied from `#retrieve` above.

https://github.com/HiMamaInc/payrix/blob/a0934b4dbe6e87409becf580bc349d3bff724245/lib/payrix/resource/base.rb#L90-L101

Reply:
Could you clarify why we need this while creating a resource? Not sure I fully comprehend the intention.

Reply:
This allows us to return related fields in the response after creating a resource. We need this in order to inspect the `txnResults`, which contain an errors (error codes, messages) and to inspect the `payment`, which gives us details about which type of credit card was used, so we can calculate the estimated fees paid.

Reply:
Okay, thanks for clarifying. I guess not making these changes would mean querying the `txnResults` and `payment` separately?